integrations/extensions/starter-kits/neuralseek/neuralseek-actions.json

```diff
@@ -212,25 +206,82 @@
             },
             "handlers": [],
             "resolver": {
-              "type": "end_action"
+              "type": "continue"
             },
             "variable": "step_949",
             "condition": {
               "and": [
                 {
-                  "exists": {
-                    "variable": "step_519_result_1",
-                    "variable_path": "body.url"
-                  }
+                  "contains": [
+                    {
+                      "variable": "step_519_result_1",
+                      "variable_path": "body.url"
+                    },
+                    {
+                      "scalar": "http",
+                      "options": {
+                        "ignore_case": true
+                      }
+                    }
+                  ]
                 },
                 {
-                  "expression": "${step_519_result_1}.body.score > 0.1"
+                  "gt": [
+                    {
+                      "variable": "step_519_result_1",
+                      "variable_path": "body.score"
+                    },
+                    {
+                      "scalar": "10"
+                    }
+                  ]
                 }
               ]
-            }
+            },
+            "next_step": "step_444"
+          },
+          {
+            "step": "step_444",
+            "output": {
+              "generic": [
+                {
+                  "values": [
+                    {
+                      "text_expression": {
+                        "concat": [
+                          {
+                            "scalar": "Rate this answer:"
+                          }
+                        ]
+                      }
+                    }
+                  ],
+                  "response_type": "text",
+                  "selection_policy": "sequential"
+                },
+                {
+                  "title": "",
+                  "source": "${step_519_result_1.body.thumbs}?style=background-color%3A%23f4f4f4%3B",
+                  "response_type": "iframe",
+                  "channel_options": {
+                    "chat": {
+                      "display": "inline",
+                      "dimensions": {
+                        "base_height": "45"
```

Comment:
Do we wanna make this taller so we don't get the scrollbars?

Reply:
I tried to recreate the scrollbar, but I can't -- I tried in multiple browsers, resizing the window, etc., and it doesn't re-appear.  So I just updated the sample screenshot and am leaving this value as is.